docs/remote/containers-advanced.md

```diff
@@ -421,17 +447,15 @@ Follow these steps:
     ssh -NL localhost:23750:/var/run/docker.sock user@hostname
     ```
 
-3. Next, use the **local** SSHFS mount to open `~/devcontainers/repository-name-here`.
+3. Next, use the **local** SSHFS mount to open `~/devcontainers/repository-name-here` in VS Code.
 
-4. Update `docker.host` to the appropriate value in workspace settings (`.vscode/settings.json`) to point to the SSH tunnel we will start up.
+4. Update `docker.host` to the appropriate value in workspace settings (`.vscode/settings.json`) to point to the SSH tunnel we started.
 
     ```json
-    {
-      "docker.host":"tcp://localhost:23750"
-    }
+    "docker.host":"tcp://localhost:23750"
     ```
 
-5. Next, use the file mount to copy `~/repos/your-repository-name/.devcontainer/devcontainer.json` to `.devcontainer.json` (dot-prefixed) in `~/devcontainers/repository-name-here` and make a few edits. In this example, we are using a Dockerfile, so the changes would be as follows:
+5. Next, use the file mount to copy `~/repos/your-repository-name/.devcontainer/devcontainer.json` to `.~~devcontainer~~.json` (dot-prefixed) in `~/devcontainers/repository-name-here` and update the default workspace mount in the file to point to the source code on the remote machine. In this example, we are using a Dockerfile, so the changes would be as follows:
```

Comment:
The double `~~` look out of place, are these Markdown or typos?

Reply:
Thanks - Yeah "ctrl+s" on mac does that with markdown all-in-one extension installed.  Must have been jumping between my windows and mac at the time.